modular_skyrat/modules/ashwalkers/code/buildings/railroad.dm

```diff
@@ -112,6 +113,15 @@
 	. = ..()
 	atom_storage?.show_contents(user)
 
+/obj/vehicle/ridden/rail_cart/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/stack/ore/glass))
+		var/obj/item/stack/ore/glass/use_item = I
+		if(GetComponent(/datum/component/simple_farm) || !use_item.use(10))
+			return ..()
+		AddComponent(/datum/component/simple_farm, TRUE, TRUE, list(0, 16))
+		return
+	return ..()
```

Comment:
Single-letter var bad

Also, uh... Does that make the minecart stationary?
And if not, can it still be ridden? I feel like it'd probably be a bit awkward if it can still be ridden when there's also something inside of it.

Also, would like to see a screenshot of what that looks like in-game!

Reply:
I'm an idiot there was one, I just didn't check 🧠